packages/design-system/src/styles/components/_Table.scss

```diff
@@ -83,6 +83,7 @@
   text-align: left;
 }
 
+// ❓ Do we need these text-align classes when we have utils that do the same thing?
```

Comment:
I vote to use the more generic alignment classes and to deprecate these.

Reply:
Oh yeah, that's a great idea!